wkdehdlr/CoinHelper/app/src/main/java/kr/schoolsharing/coinhelper/tasks/UpbitFragment.kt

```diff
@@ -50,8 +22,9 @@ class UpbitFragment : Fragment(), UpbitContract.View {
 
         //TODO : MARKET_NAME enum으로 빼자
         val marketName = arguments?.get("MARKET_NAME").toString()
-        presenter = UpbitPresenter(Repository, this)
-        presenter.start(marketName)
+        UpbitViewModel(Repository).apply {
```

Comment:
다음과 같이 사용하는 것이 좋을 것 같습니다.
```suggestion
        binding.viewModel = UpbitViewModel(Repository).apply {
            loadUpbitMarket(marketName)
        }
```

Reply:
[8df16c6](https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/210/commits/8df16c65176af7a83b6e1ac0789bc77681fdb063) 수정했습니다